types/sanctuary/index.d.ts

```diff
@@ -177,6 +181,12 @@ declare namespace Sanctuary {
     I<A>(x: A): A;
     K<A>(x: A): (y: any) => A;
     T<A>(x: A): <B>(f: Fn<A, B>) => B;
+    // Pair
+    Pair<A>(a: A): <B>(b: B) => Pair<A, B>;
+    pair<A, B, C>(f: Fn2<A, B, C>): (p: Pair<A, B>) => C;
+    fst<A>(p: Pair<A, any>): A;
+    snd<B>(p: Pair<any, B>): B;
```

Comment:
```suggestion
    fst<A, B>(p: Pair<A, B>): A;
    snd<A, B>(p: Pair<A, B>): B;
```

Would the above work equally well?

Reply:
I tried that at first, but the linter in the repo CI throwns an error about unused variables 😞 
I can check if these linter rules are global of if we can adjust them per project

Reply:
made the adjustments! in the next PRs I'm going to look at `zip` declarations
it looks like that the upcoming ts version will make it easier to declare some expressions https://devblogs.microsoft.com/typescript/announcing-typescript-4-0-beta/